services/meta/data.go

```diff
@@ -465,8 +466,33 @@ func (data *Data) DropContinuousQuery(database, name string) error {
 	return ErrContinuousQueryNotFound
 }
 
+// validateURL returns an error if the URL does not have a port or uses a scheme other than UDP or TCP.
+func validateURL(input string) error {
+	u, err := url.Parse(input)
+	if err != nil {
+		fmt.Println(err)
+		return ErrInvalidSubscriptionURL(input)
+	}
+
+	if u.Scheme != "udp" && u.Scheme != "tcp" {
+		return ErrInvalidSubscriptionURL(input)
+	}
+
+	hostParts := strings.Split(u.Host, ":")
```

Comment:
I'm not sure what the intention is here. Are you checking that a port has been set on the URL?